app/models/donor.rb

```diff
@@ -11,6 +11,8 @@ class Donor < ActiveRecord::Base
 
   scope :search, ->(keyword) { where("name ILIKE :keyword OR identification ILIKE :keyword", keyword: "%#{keyword}%") }
 
+  scope :same_postcode, ->(postcode) { where("postal_code ILIKE ?", postcode) }
```

Comment:
How does `ILIKE` work on a string of numbers? Can we just do strict equality instead, or does that miss some?